### DISCUSSION TITLE
Usage example without React

### DISCUSSION THREAD

**Discussion by cheeaun** [uber/hubble.gl#22] [Q&A] [answered]
On the [site](https://hubble.gl/docs), there's example code on how to implement this with React.

Personally I use deck.gl without React, and I think it would be great to have documentation on how to implement this **without** React.

Thanks! 🙇

**Comment by chrisgervang** [comment]
Hey thanks for the question! And apologies for the delayed response. We've been recently focused on adding hubble.gl to react applications, like kepler.gl, so that non-programers can utilize the tool. Once this is complete I will be shifting focus back into the developer experience for programmers. A non-react example is definitely something we can do in the future.

I'm also available to help you debug if you take a crack at it via CodePen. Hubble should nearly work out of the box without React since its core doesn't depend on React functionality.

**Comment by benliq** [comment]
Hi thanks for the amazing tool @chrisgervang .
When are you planning to be able to release this new implementation? Will it include a working example based on Kepler?
Thanks a lot!
Ben

**Comment by chrisgervang** [comment]
Hey @benliq you can monitor progress on our kepler integration in https://github.com/uber/hubble.gl/pull/34 - it's getting close to an proof of concept, and we'll be flushing it out to include in kepler.gl by default.

**Comment by chrisgervang** [comment, marked as answer]
Pure JS support added with 1.3.0! Check out https://github.com/visgl/hubble.gl/tree/master/examples/pure-js


https://user-images.githubusercontent.com/2461547/137951732-4c615b00-5d12-4b24-a056-20ae82ca8ce0.mov
